lbrynet/p2p/BlobManager.py

```diff
@@ -120,6 +127,7 @@ def delete_blobs(self, blob_hashes):
     @defer.inlineCallbacks
     def _completed_blobs(self, blobhashes_to_check):
         """Returns of the blobhashes_to_check, which are valid"""
+        # This iterative method might not be the best since get_blob raises TypeError
```

Comment:
Please open an issue explaining what needs to be fixed / review with a permalink to relevant lines of code instead of a comment here.

Reply:
That comment wasn't supposed to go into the to-be-pulled, it was supposed to be part of something completely different.